proto/lavanet/lava/rewards/query.proto

```diff
@@ -63,4 +69,21 @@ message QueryBlockRewardResponse {
   cosmos.base.v1beta1.Coin reward = 1 [(gogoproto.nullable) = false];
 }
 
+// QueryProviderRewardRequest is request type for the Query/ProviderReward RPC method.
+message QueryProviderRewardRequest {
+  string chainID  = 1;
```

Comment:
use snake case (not camel case) as we do in all protos. i.e. chainID -> chain_id